beta/CONTRIBUTING.md

```diff
@@ -1,50 +1,50 @@
-# Contributing
+# Contributing {/*contributing*/}
```

Comment:
What's this?

Reply:
Seems like lint staged ran for `contributing.md` since all md files have a generateHeading linter. We might want to add readme and contributing.md to lint staged ignore.

Reply:
lol. it's our post-commit hook getting antsy because i edited this file. i'll undo manually